tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -3177,13 +3183,14 @@ void SPIRVEmitter::storeValue(const SpirvEvalInfo &lhsPtr,
       const auto *field = cast<FieldDecl>(decl);
       assert(field);
 
-      const auto subRhsValType =
-          typeTranslator.translateType(field->getType(), rhsVal.layoutRule);
+      const auto subRhsValType = typeTranslator.translateType(
+          field->getType(), rhsVal.getLayoutRule());
       const auto subRhsVal =
           theBuilder.createCompositeExtract(subRhsValType, rhsVal, {index});
       const auto subLhsPtrType = theBuilder.getPointerType(
-          typeTranslator.translateType(field->getType(), lhsPtr.layoutRule),
-          lhsPtr.storageClass);
+          typeTranslator.translateType(field->getType(),
+                                       lhsPtr.getLayoutRule()),
+          lhsPtr.getStorageClass());
```

Comment:
bad formatting here.

Reply:
Also will fix in the next commit

Reply:
Well, actually it is not. ClangFormat is happy with this.

Reply:
lol, it seems like an odd one! hah
Ok thanks